internal/service/encryptionatrest/resource_migration_test.go

```diff
@@ -142,23 +136,14 @@ func TestMigEncryptionAtRest_basicAzure(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				ExternalProviders: mig.ExternalProviders(),
-				Config:            acc.ConfigEARAzureKeyVault(projectID, &azureKeyVault, false, false), // not using data source as it was introduced in 1.19.0
+				Config:            acc.ConfigEARAzureKeyVault(projectID, &azureKeyVault, false, useDatasource), // not using data source as it was introduced in 1.19.0
```

Comment:
knit: remove comment as info is already in the useDatasource var assignment

Reply:
updated